docs/fundamentals/syslib-diagnostics/syslib0047.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SYSLIB0047 warning - XmlSecureResolver is obsolete
+description: Learn about the obsoletion of XmlSecureResolver that generates compile-time warning SYSLIB0047.
+ms.date: 04/08/2022
+---
+# SYSLIB0047: XmlSecureResolver is obsolete
+
+The <xref:System.Xml.XmlSecureResolver?displayProperty=fullName> type is obsolete, starting in .NET 7. Using it in code generates warning `SYSLIB0047` at compile time.
+
+## Workaround
+
+Consider using the static property `XmlResolver.ThrowingResolver` instead. This property provides an <xref:System.Xml.XmlResolver> instance that forbids external entity resolution.
```

Comment:
Is there not an xref for this `XmlResolver.ThrowingResolver` yet?

Reply:
No, it will appear after RC 1.